src/components/Gallery/GalleryItem/gallery-item.scss

```diff
@@ -49,7 +49,7 @@
 
   &.image-selected {
     .date-time {
-      border-top: 3px solid $active-green;
+      border-top: 3px solid #f29518;
```

Comment:
I'm not sure of the relevance of my review, but you can add a new variable (or use an existing one) in `variables.scss` to keep a coherence for this project

Reply:
Done!